WORKSPACE

```diff
@@ -31,7 +31,7 @@ go_rules_dependencies()
 
 go_register_toolchains(
     nogo = "@//hack/build:nogo_vet",
-    version = "1.15",
+    version = "1.16",
```

Comment:
```suggestion
    version = "1.16.2",
```

Reply:
See https://golang.org/dl/ and https://golang.org/doc/devel/release.html#go1.16

Reply:
Thank you, that makes sense!

I _think_ that you cannot specify patch version in `go.mod` file, see parsing [here](https://github.com/golang/mod/blob/master/modfile/rule.go#L211).
If I try to do that locally and run i.e `go mod tidy` it will fail with `.. must match format 1.23`. 
As I understand that directive just sets the minimum required version of go, so perhaps it's okay to be `1.16`.

For the toolchain it makes sense I guess since that actually downloads `go`. 
I've added the change, please take a look :)